test/cases/getSavedTimeForUser.js

```diff
@@ -5,8 +5,8 @@ var getHash = require('../../src/utils/getHash.js');
 
 describe('getSavedTimeForUser', () => {
   before(() => {
-    let startOfQuery = "INSERT INTO sponsorTimes (videoID, startTime, endTime, votes, UUID, userID, timeSubmitted, views, category, shadowHidden) VALUES";
-    db.exec(startOfQuery + "('getSavedTimeForUser', 1, 11, 2, 'abc1239999', '" + getHash("testman") + "', 0, 50, 'sponsor', 0)");
+    let startOfQuery = "INSERT INTO sponsorTimes (videoID, startTime, endTime, votes, UUID, userID, timeSubmitted, views, category, shadowHidden, hashedVideoID) VALUES";
+    db.exec(startOfQuery + "('getSavedTimeForUser', 1, 11, 2, 'abc1239999', '" + getHash("testman") + "', 0, 50, 'sponsor', 0, '" + getHash('getSavedTimeForUser', 0) + "')");
```

Comment:
`getSavedTimeForUser` is confusing for a videoID. Maybe choose something more obivous.

Reply:
I am just hashing the name that was already in the test data. Someone that came before me picked `getSavedTimeForUser` as the videoID value for the database row.

Should I change nonsensical videoID values like this one whenever I see them in tests? What about those that include things like commas?

Reply:
Oh sorry, I didn't notice this was actually a test for `getSavedTimeForUser`, I guess I understand why the name is like that. You can leave it.

Reply:
> Should I change nonsensical videoID values like this one whenever I see them in tests? What about those that include things like commas?
The commas could be to test to make sure the application is able to deal with commas in the data (a , isn't a valid character in a video id but _ and - are for example)